eng/pipelines/templates/jobs/ci.yml

```diff
@@ -225,36 +273,62 @@ jobs:
           arguments: 'restore'
           workingDirectory: '$(Agent.BuildDirectory)'
 
-      - task: PowerShell@2
-        displayName: "Verify generated code"
+      - pwsh: |
+          $failed = $false
+          if ("$(ProjectNames)") {
+            $serviceDirectories = "$(ChangedServices)" -split ","
+            $ScanPaths = @()
+            foreach ($ServiceDirectory in $serviceDirectories) {
```

Comment:
Seems like we should parallelize this, either via devops jobs or just via powershell jobs. Looks like it takes 90 seconds for core, not sure about the others.

Reply:
I will in a follow-up PR.